spec/features/admin_dashboard_driver_errors_spec.rb

```diff
@@ -18,9 +18,19 @@
     visit admin_root_path
 
     click_on "Driver Errors"
+
+    expect(page).to have_link(
+      driver_error.driver_application.snap_application_id,
+    )
```

Comment:
I'm not sure if there is a more preferrable way to assert this link exists.

Reply:
this seems good to me!